src/Primary_Extension/org/lobobrowser/primary/ext/TextViewerWindow.java

```diff
@@ -43,6 +49,10 @@
   private final JTextArea textArea;
   private boolean scrollsOnAppends;
 
+  // Key Stroke for CMD in case of Mac and CTRL otherwise
+  private static final int CMD_CTRL_KS = PlatformInit.OS_NAME == OS.MAC ? Toolkit.getDefaultToolkit().getMenuShortcutKeyMask()
```

Comment:
Since this mask is used in multiple places, you could define it in `PlatformInit` and call it `CMD_CTRL_KS_MASK`.